crates/conjure_core/src/solver/adaptors/common.rs.txt

```diff
@@ -0,0 +1,45 @@
+use crate::solver::SolverFamily;
```

Comment:
Why is this a .txt file? Doesn't that mean it will get ignored?

Reply:
This is not a file that I have created but I have removed the .txt from the file for now. Shikhar and I are going to sit and clean the files this week.